### PR TITLE
Domains: Add partial Transfer Domain test

### PR DESCRIPTION
**Pull request by aidvu**
- Should be able to select transfer screen
- Should be able to get to precheck screen
- Make sure there's an auth code error

Fixes: https://github.com/Automattic/wp-e2e-tests/issues/1353

### CODE REVIEW

File: specs/wp-manage-domains-spec.js [hunk 1]
```diff
@@ -185,4 +187,84 @@ describe( `[${ host }] Managing Domains: (${ screenSize }) @parallel`, function(
 			return await shoppingCartWidgetComponent.empty();
 		} );
 	} );
+
+	describe( 'Transfer a domain to an existing site (partial) @parallel', function() {
+		const domain = 'andrija.me';
```
Comment (bsessions85) [inline specs/wp-manage-domains-spec.js]: I think this should probably be put in to the encrypted config file

File: specs/wp-manage-domains-spec.js [hunk 2]
```diff
@@ -185,4 +187,84 @@ describe( `[${ host }] Managing Domains: (${ screenSize }) @parallel`, function(
...
+		} );
+
+		step( 'Click check auth code', async function() {
+			const transferDomainPrecheckPage = await TransferDomainPrecheckPage.Expect( driver );
+			return await transferDomainPrecheckPage.clickCheckAuthCode();
+		} );
+
+		step( 'Can see auth code error', async function() {
+			const transferDomainPrecheckPage = await TransferDomainPrecheckPage.Expect( driver );
+			return await transferDomainPrecheckPage.clickCheckAuthCode();
```
Comment (bsessions85) [inline specs/wp-manage-domains-spec.js]: Should this 'authCodeErrorDisplayed?

File: specs/wp-manage-domains-spec.js (hunk 1 above)
Comment (aidvu) [reply specs/wp-manage-domains-spec.js]: This is just a random taken domain. Nothing important in any way. :)

File: specs/wp-manage-domains-spec.js (hunk 2 above)
Comment (aidvu) [reply specs/wp-manage-domains-spec.js]: 🤦‍♂️

**Pull request merged**